### PR TITLE
Improve UI flow online data sources/small_Fixes

### PR DESCRIPTION
**Pull request by Tristan611**
### Checklist

- [X] Tests have been written/updated (if necessary)
- [X] All tests are passing
- [X] Docs have been updated (if necessary)
- [X] Updated `Linear` issue to `In Review`

### Description

In this pull request the final updates are going to be added to the developer branche.

- we have made the new home screen with the floating action button. 
- Removed the devices screen.
- Improved the UI flow for adding screens and adding online data sources.

### Linked Issues

https://trello.com/c/b5J6FgK4

### CODE REVIEW

File: apps/expo/package.json [hunk 1]
```diff
@@ -54,13 +54,15 @@
...
     "react-native-esp-idf-provisioning": "ssh://git@github.com/energietransitie/NeedForHeat-esp-idf-provisioning",
+    "react-native-exit-app": "^1.1.0",
     "react-native-gesture-handler": "2.8.0",
     "react-native-permissions": "3.6.1",
     "react-native-prompt-android": "1.1.0",
     "react-native-reanimated": "2.12.0",
     "react-native-render-html": "6.3.4",
     "react-native-safe-area-context": "4.4.1",
     "react-native-screens": "3.18.0",
+    "react-native-stop-app": "^0.1.31",
```
Comment (jortdr) [inline apps/expo/package.json:65]: remove

File: apps/expo/src/screens/home/AddOnlineDataSourceScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    const { theme } = useTheme();
+    const [loading, setLoading] = useState(false);
+    const { openUrl } = useOpenExternalLink();
+
+    const onCancel = () => {
+        navigation.goBack();
+    }
+
+    const onConnectEnelogic = async () => {
+        //do something
```
Comment (jortdr) [inline apps/expo/src/screens/home/AddOnlineDataSourceScreen.tsx:30]: remove

File: apps/expo/src/screens/home/DeviceOverviewScreen/index.tsx [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+  const buildings = user?.buildings ?? [];
+  const [buildingId, setBuildingId] = useState<number | undefined>(buildings[0]?.id);
+
+  const hasMultipleBuildings = buildings.length > 1;
+
+
+  //Enelogic stuff
+  const [loading, setLoading] = useState(false);
+  //https://enelogic.com/oauth/v2/auth?response_type=code&client_id=10321_iee4evob2rcw4scc4wcwos448ogcwkgocoswwwkkow4wcckk4&redirect_uri=nfh%3A%2F%2Fcallback%3Fprovider%3D1&scope=account&state=TODO
```
Comment (jortdr) [inline apps/expo/src/screens/home/DeviceOverviewScreen/index.tsx:35]: remove

File: apps/expo/src/screens/settings/ExternalProviderScreen/_externalProviderListItem.tsx [hunk 4]
```diff
@@ -30,6 +30,8 @@ export default function ExternalProviderListItem(props: ExternalProviderListItem
       item.cloud_feed.scope
     )}&state=${state}`;
 
+    console.log("redirectUrl", redirectUrl);
```
Comment (jortdr) [inline apps/expo/src/screens/settings/ExternalProviderScreen/_externalProviderListItem.tsx:33]: remove

File: apps/expo/src/screens/settings/SettingsScreen/index.tsx [hunk 5]
```diff
@@ -85,15 +85,15 @@ export default function SettingsScreen() {
...
-      : []),
+    // ...(isAuthed
+    //   ? [
+    //       {
+    //         title: t("screens.settings_stack.settings_screen.external_providers"),
+    //         value: "",
+    //         onPress: () => settingsNavigation.navigate("ExternalProviderScreen"),
+    //       },
+    //     ]
+    //   : []),
```
Comment (jortdr) [inline apps/expo/src/screens/settings/SettingsScreen/index.tsx:96]: remove

File: apps/expo/src/components/common/ManualContent/index.tsx [hunk 6]
```diff
@@ -30,19 +30,22 @@ export default function ManualContent({ manualUrl, languageHeader }: ManualConte
       contentContainerStyle={{
         alignItems: "center",
         minHeight: "90%",
+        marginLeft: "3%",
+        paddingRight: "4%",
+        paddingLeft: "7%",
```
Comment (jortdr) [inline apps/expo/src/components/common/ManualContent/index.tsx:35]: Not sure if these will fit for iOS

File: apps/expo/src/components/common/ManualContent/index.tsx [hunk 7]
```diff
@@ -12,7 +12,7 @@ type ManualContentProps = {
 };
 
 export default function ManualContent({ manualUrl, languageHeader }: ManualContentProps) {
-  const [onLayout, width] = useLayoutWidth();
+  // const [onLayout, width] = useLayoutWidth();
```
Comment (jortdr) [inline apps/expo/src/components/common/ManualContent/index.tsx:15]: rm -rf

**Review by jortdr — APPROVED** [review]
LGTM

**Comment by NormalNickYT** [comment]
Nice work boys!

**Pull request merged**